<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mplayer_video</class>
 <widget class="QMainWindow" name="mplayer_video">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>801</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>801</width>
    <height>507</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mplayer_video</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>801</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>801</width>
     <height>450</height>
    </size>
   </property>
   <widget class="QPushButton" name="open">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>601</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="zanting">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quick">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>快进</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>回退</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prio">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>上一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>下一个</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="percentLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="voice">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>377</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="voice_slider">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>181</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="del">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>播放列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jiansu">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>减速</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jiasu">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>加速</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jietu">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>截屏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lupin">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color:rgb(103, 103, 103); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 69, 234); }</string>
    </property>
    <property name="text">
     <string>录屏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lupinstart">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="lupin_lable">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_bofang">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>380</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
